src/main/java/uk/gov/hmcts/opal/launchdarkly/FeatureToggleApi.java

```diff
@@ -43,13 +42,17 @@ public boolean isFeatureEnabled(String feature, LDUser user, boolean defaultValu
     }
 
     public String getFeatureValue(String feature, String defaultValue) {
-        return internalClient.stringVariation(feature, createLDContext().build(), defaultValue);
+        return internalClient.stringVariation(feature, getContext(), defaultValue);
     }
 
-    public ContextBuilder createLDContext() {
-        return LDContext.builder("opal")
-            .set("timestamp", String.valueOf(System.currentTimeMillis()))
-            .set("environment", environment);
+    public LDUser.Builder createLDUser() {
+        return new LDUser.Builder("opal")
+            .custom("timestamp", String.valueOf(System.currentTimeMillis()))
+            .custom("environment", environment);
+    }
+
+    public LDContext getContext() {
+        return LDContext.fromUser(createLDUser().build());
```

Comment:
The LDContext.fromUser is deprecated. I had made a change this morning - but maybe what I did was wrong?